v0/destinations/campaign_manager/transform.js

```diff
@@ -60,15 +62,19 @@ function processTrack(message, metadata, destination) {
     message,
     mappingConfig[ConfigCategories.TRACK.name]
   );
-  requestJson.limitAdTracking =
-    requestJson.limitAdTracking || destination.Config.limitAdTracking;
+  requestJson.nonPersonalizedAd =
+      requestJson.nonPersonalizedAd != null ? requestJson.nonPersonalizedAd : destination.Config.nonPersonalizedAd;
   requestJson.treatmentForUnderage =
-    requestJson.treatmentForUnderage || destination.Config.treatmentForUnderage;
+      requestJson.treatmentForUnderage != null ? requestJson.treatmentForUnderage : destination.Config.treatmentForUnderage;
   requestJson.childDirectedTreatment =
-    requestJson.childDirectedTreatment ||
-    destination.Config.childDirectedTreatment;
-  requestJson.nonPersonalizedAd =
-    requestJson.nonPersonalizedAd || destination.Config.nonPersonalizedAd;
+      requestJson.childDirectedTreatment != null ? requestJson.childDirectedTreatment : destination.Config.childDirectedTreatment;
+  requestJson.limitAdTracking =
+      requestJson.limitAdTracking != null ? requestJson.limitAdTracking : destination.Config.limitAdTracking;
+  // updating these values is not allowed
```

Comment:
Can we use `||` only?

Reply:
if we use or , then if its true in config and false in properties.traits, preference will be given to config not traits.

Reply:
You can use `isDefinedAndNotNull` to check then rather than using `!=null` check